forward/testing/spike_test.lua

```diff
@@ -0,0 +1,68 @@
+local Rewriting = require("rewriting")
+local godefs = require("godefs")
+local C = require("ffi").C
+local Datagram = require("lib.protocol.datagram")
+local Ethernet = require("lib.protocol.ethernet")
+local IPV4 = require("lib.protocol.ipv4")
+local TCP = require("lib.protocol.tcp")
+
+local P = require("apps.pcap.pcap")
+local link = require("core.link")
+local pcap = require("testing/pcap")
```

Comment:
Do we really need to process pcap files?  Spike won't be running on pcaps in the end, so I think it's unnecessary to test the pcap file handling.

Reply:
I just copied in Snabb's pcap file handler as a temporary solution to write the output packets to pcap (so that we can see them). We can remove it when the testing framework matures beyond manually checking tests.

Reply:
Any reason not to just `require("lib.pcap.pcap")` or something similar?

Reply:
I tried that earlier but it didn't work. I figured that it wasn't part of the exposed API, but it might have been a mistake on my end.